packages/core/src/util/concurrency.ts

```diff
@@ -49,31 +49,29 @@ export type FunctionType<T> = (
  * @param collection array of elements to be processed
  * @param fn the function to be executed asynchronously
  * @param options additional settings
```

Comment:
nit: in my C++ days, we would put the "preconditions" in the function header, so the calling programmer understand how to use the function at a glance - without needing to read the code 

for example, in this case we fail on different conditions in the `minJitterMillis` checks, but don't mention it in the header

Reply:
I documented it on its own type so when you hover the mouse over it you see the docs for the property (only adding to the function it doesn't work). I think that's the right place to add it, so we also are not duplicating stuff - worse than no doc is stale/wrong doc.

Cool?